
class MyException(Exception):
    pass


class CountException(MyException):
    pass